### DISCUSSION TITLE
Tiny POP [UK]

### DISCUSSION THREAD

**Discussion by dantheman121** [iptv-org/community#901] [Q&A] [answered]
### Channel ID

TinyPop.uk

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by ExperiencersInternational** [comment, marked as answer]
There was a broadcastcdn link, but it doesn't seem to be working right now, I've only gotten those links to work with Channel 4, Channel 5, BBC News and E4.

![Screenshot_2023-04-30-23-15-40-390_com microsoft emmx](https://user-images.githubusercontent.com/56035537/235378545-569f929b-b979-4e6f-9386-a5afc1e1c297.jpg)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/uk.m3u#L181-L186
